src/flamenco/runtime/program/fd_bpf_loader_v3_program.c

```diff
@@ -311,23 +334,13 @@ common_close_account( fd_pubkey_t * authority_address,
     return FD_EXECUTOR_INSTR_ERR_MISSING_REQUIRED_SIGNATURE;
   }
 
-  fd_borrowed_account_t * close_account = NULL;
-  int err = fd_instr_borrowed_account_modify_idx( instr_ctx, 0UL, 0UL, &close_account );
-  if( FD_UNLIKELY( err!=FD_ACC_MGR_SUCCESS ) ) {
-    FD_LOG_WARNING(( "Borrowed account lookup for close account failed" ));
-    return err;
-  }
-  fd_borrowed_account_t * recipient_account = NULL;
-  err = fd_instr_borrowed_account_modify_idx( instr_ctx, 1UL, 0UL, &recipient_account );
-  if( FD_UNLIKELY( err!=FD_ACC_MGR_SUCCESS ) ) {
-    FD_LOG_WARNING(( "Borrowed account lookup for recipient account failed" ));
-    return err;
-  }
+  FD_BORROWED_ACCOUNT_TRY_BORROW_IDX( instr_ctx, 0UL, close_account     ) {
```

Comment:
Could we use this macro in all the native programs? 🙏

Reply:
not ready for review yet